platforms/software/testing-base/src/main/java/org/gradle/api/tasks/testing/AbstractTestTask.java

```diff
@@ -318,18 +318,7 @@ public void removeTestOutputListener(TestOutputListener listener) {
      */
     @Internal
     @Override
-    @ToBeReplacedByLazyProperty
-    public boolean getIgnoreFailures() {
-        return ignoreFailures;
-    }
-
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public void setIgnoreFailures(boolean ignoreFailures) {
-        this.ignoreFailures = ignoreFailures;
-    }
+    public abstract Property<Boolean> getIgnoreFailures();
```

Comment:
ℹ️  Have to keep the override here to preserve compatibility, because it changes `@Input`=> `@Internal` for this property

At the same time, it seems like a semantic mistake, since we use the value of this property at task execution time here:
https://github.com/gradle/gradle/blob/6254fa10c52451d580856eb3ea72205d85836b08/platforms/software/testing-base/src/main/java/org/gradle/api/tasks/testing/AbstractTestTask.java#L699

@gradle/bt-jvm, should this stay `@Internal` or should it become an input?